docs/reference/commandline/update.md

```diff
@@ -21,12 +21,13 @@ Usage:  docker update [OPTIONS] CONTAINER [CONTAINER...]
 Update configuration of one or more containers
 
 Options:
-      --blkio-weight value          Block IO (relative weight), between 10 and 1000
+      --blkio-weight uint16         Block IO (relative weight), between 10 and 1000, or 0 to disable (default 0)
```

Comment:
Thanks @thaJeztah. I take a look. To fix this issue may require changes in pflag part which may involve other changes. I will create another PR to address this issue.